2022-11-21 03:33:56,378 INFO    MainThread:19568 [wandb_setup.py:_flush():68] Configure stats pid to 19568
2022-11-21 03:33:56,378 INFO    MainThread:19568 [wandb_setup.py:_flush():68] Loading settings from C:\Users\user\.config\wandb\settings
2022-11-21 03:33:56,378 INFO    MainThread:19568 [wandb_setup.py:_flush():68] Loading settings from c:\Users\user\Documents\GitHub\ict3104-team09-2022-binder\Codes\TSU\Toyota_Smarthome\pipline\wandb\settings
2022-11-21 03:33:56,378 INFO    MainThread:19568 [wandb_setup.py:_flush():68] Loading settings from environment variables: {'_require_service': 'True'}
2022-11-21 03:33:56,378 INFO    MainThread:19568 [wandb_setup.py:_flush():68] Inferring run settings from compute environment: {'program_relpath': 'Codes\\TSU\\Toyota_Smarthome\\pipline\\train.py', 'program': 'c:\\Users\\user\\Documents\\GitHub\\ict3104-team09-2022-binder\\Codes\\TSU\\Toyota_Smarthome\\pipline\\train.py'}
2022-11-21 03:33:56,417 INFO    MainThread:19568 [wandb_init.py:_log_setup():476] Logging user logs to c:\Users\user\Documents\GitHub\ict3104-team09-2022-binder\Codes\TSU\Toyota_Smarthome\pipline\wandb\run-20221121_033356-170aqg4h\logs\debug.log
2022-11-21 03:33:56,418 INFO    MainThread:19568 [wandb_init.py:_log_setup():477] Logging internal logs to c:\Users\user\Documents\GitHub\ict3104-team09-2022-binder\Codes\TSU\Toyota_Smarthome\pipline\wandb\run-20221121_033356-170aqg4h\logs\debug-internal.log
2022-11-21 03:33:56,418 INFO    MainThread:19568 [wandb_init.py:_jupyter_setup():426] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x0000017B0C206CC0>
2022-11-21 03:33:56,420 INFO    MainThread:19568 [wandb_init.py:init():516] calling init triggers
2022-11-21 03:33:56,420 INFO    MainThread:19568 [wandb_init.py:init():520] wandb.init called with sweep_config: {}
config: {}
2022-11-21 03:33:56,420 INFO    MainThread:19568 [wandb_init.py:init():569] starting backend
2022-11-21 03:33:56,420 INFO    MainThread:19568 [wandb_init.py:init():573] setting up manager
2022-11-21 03:33:56,428 INFO    MainThread:19568 [backend.py:_multiprocessing_setup():104] multiprocessing start_methods=spawn, using: spawn
2022-11-21 03:33:56,435 INFO    MainThread:19568 [wandb_init.py:init():580] backend started and connected
2022-11-21 03:33:56,448 INFO    MainThread:19568 [wandb_run.py:_label_probe_notebook():1113] probe notebook
2022-11-21 03:33:56,448 INFO    MainThread:19568 [wandb_run.py:_label_probe_notebook():1123] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2022-11-21 03:33:56,448 INFO    MainThread:19568 [wandb_init.py:init():658] updated telemetry
2022-11-21 03:33:56,485 INFO    MainThread:19568 [wandb_init.py:init():694] communicating run to backend with 60 second timeout
2022-11-21 03:33:57,399 INFO    MainThread:19568 [wandb_run.py:_on_init():2000] communicating current version
2022-11-21 03:33:57,438 INFO    MainThread:19568 [wandb_run.py:_on_init():2004] got version response 
2022-11-21 03:33:57,438 INFO    MainThread:19568 [wandb_init.py:init():728] starting run threads in backend
2022-11-21 03:33:57,592 INFO    MainThread:19568 [wandb_run.py:_console_start():1980] atexit reg
2022-11-21 03:33:57,592 INFO    MainThread:19568 [wandb_run.py:_redirect():1838] redirect: SettingsConsole.WRAP_RAW
2022-11-21 03:33:57,593 INFO    MainThread:19568 [wandb_run.py:_redirect():1903] Wrapping output streams.
2022-11-21 03:33:57,593 INFO    MainThread:19568 [wandb_run.py:_redirect():1925] Redirects installed.
2022-11-21 03:33:57,593 INFO    MainThread:19568 [wandb_init.py:init():765] run started, returning control to user process
2022-11-21 03:34:00,101 INFO    MainThread:19568 [wandb_init.py:_pause_backend():389] pausing backend
2022-11-21 03:34:00,101 INFO    MainThread:19568 [jupyter.py:save_ipynb():378] not saving jupyter notebook
2022-11-21 03:35:22,652 INFO    MainThread:19568 [wandb_init.py:_resume_backend():398] resuming backend
2022-11-21 03:35:26,081 INFO    MainThread:19568 [wandb_init.py:_pause_backend():389] pausing backend
2022-11-21 03:35:26,081 INFO    MainThread:19568 [jupyter.py:save_ipynb():378] not saving jupyter notebook
